Makefile

```diff
@@ -389,7 +389,8 @@ endef
 bundle: export RELATED_IMAGE_WASMSHIM := $(RELATED_IMAGE_WASMSHIM)
```

Comment:
I don't think `export RELATED_IMAGE_WASMSHIM := $(RELATED_IMAGE_WASMSHIM)` is required.